### DISCUSSION TITLE
How to disable the previous dates before the current date on calendar?

### DISCUSSION THREAD

**Discussion by prpanto** [shadcn-ui/ui#4544] [Q&A]
How to disable the previous dates before the current date on calendar?

**Comment by itaim18** [comment]
try something like this:
```tsx
{/* some code here*/}
   <Calendar
          mode="single"
          selected={field.value}
          onSelect={field.onChange}
          disabled={(date) =>date < new Date()}
          initialFocus
   />
{/* some code there*/}
```
lemme know how it went :)

**Reply by nadundev** [reply]
This worked thank you very much

**Reply by Siddhesh-Agarwal** [reply]
This would prevent the user from changing the date if they select an older date. This does not prevent the user from choosing an older date.
